common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/SmartcardCertPickerDialog.java

```diff
@@ -25,6 +25,7 @@
 import android.app.Activity;
 import android.content.Context;
 import android.content.DialogInterface;
+import android.nfc.Tag;
```

Comment:
👀

Is this being used?

Reply:
must have snuck in there while I was trying to log...